sdcm/nemesis.py

```diff
@@ -1548,6 +1548,66 @@ def disrupt_network_block(self):
         finally:
             self.target_node.traffic_control(None)
 
+    def disrupt_cluster_down_scaling(self):
```

Comment:
can we keep the names like `disrupt_nodetool_decommission`, i.e. `disrupt_nodetool_removenode`

Reply:
`disrupt_nodetool_removenode` or `disrupt_nodetool_terminate_and_removenode`